static/js/portico/landing-page.js

```diff
@@ -256,6 +256,4 @@ $(() => {
 // According to https://developer.mozilla.org/en-US/docs/DOM/window.onunload
 // Using this event handler in your page prevents Firefox from caching the
 // page in the in-memory bfcache (backward/forward cache).
-$(window).on("unload", () => {
-    $(window).unbind("unload");
-});
+$(window).on("unload", () => {});
```

Comment:
I suspect we should be deleting this block of code -- I think it's from some part of the ill-fated transition animation we once had on our landing pages.